Casks/p/piezo.rb

```diff
@@ -1,7 +1,13 @@
 cask "piezo" do
-  version "1.8.2"
   sha256 :no_check
 
+  on_ventura :or_older do
+    version "1.8.2"
+  end
+  on_sonoma :or_newer do
+    version "1.9.0"
+  end
```

Comment:
It seems that the system version in the livecheck url indicates the mac os version in numbers. Most recent sonoma version is 14.4.0 and so it should be separated. The version parameter do nothing I guess.

```suggestion
  on_ventura :or_older do
    version "1.8.2"

    livecheck do
      url "https://rogueamoeba.net/ping/versionCheck.cgi?format=sparkle&system=1231&bundleid=com.rogueamoeba.Piezo&platform=osx"
      strategy :sparkle
    end
  end
  on_sonoma :or_newer do
    version "1.9.0"

    livecheck do
      url "https://rogueamoeba.net/ping/versionCheck.cgi?format=sparkle&system=1440&bundleid=com.rogueamoeba.Piezo&platform=osx"
      strategy :sparkle
    end
  end
```